### DISCUSSION TITLE
Are there any plans to make a study guide for Genki II?

### DISCUSSION THREAD

**Discussion by gabriellecozart** [SethClydesdale/genki-study-resources#180] [Ideas]
Hello, I'm Gabii. I studied Japanese in college and currently want to get back into studying Japanese. I found this amazing repo while researching the differences between the 2nd and 3rd editions of the Genki books (currently have edition 2 for Genki I and II).

I am a software engineer and would love to get involved in this project if there are any needs. I've been meaning to find a side project to work on as well as study again so I think it would be a perfect fit.

**Comment by SethClydesdale** [comment]
Hello @gabriellecozart!

At the moment there are no plans to, since I'm currently occupied with working on/studying with Tobira. However, if you wish to work on/contribute a guide, you're more than welcome to 👍

**Comment by SethClydesdale** [comment]
Hello again! Just following up to let you know that I've gotten around to writing up a short study guide for Genki I & II which you can find below.

https://sethclydesdale.github.io/genki-study-resources/help/study-guide/

**Reply by trpglobokar** [reply]
~~Hey! I'm obviously not the person who originally replied, but assuming your position hasn't changed, I'd like to fork your repo and make a matching study guide for Genki II. <3 I've been trying to do some self-study w/ specifically Genki II AND do more github stuff, so this is seeming like the perfect combo project.~~

EDIT: While poking into it, I've realized that the study guide for Genki II **_is_** indeed included/implemented already. It's lessons 13-23, but somewhat easy to overlook if you don't have the chapter numbers memorized as they're not labeled "Book 1" and "Book 2" or similar in the "Quick Navigation" section. They are labeled as such in the full listings though.
